fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,47 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  setlocal enabledelayedexpansion
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (
+    echo "Active fluentdwinsvc service is detected"
+    net stop fluentdwinsvc
+  )
+  endlocal

```

Comment:
I am wondering if it is possible for the service to be active at this time.
Against my expectations, it is judged to be active in my tests.
And `net stop` often fails.

```
WixQuietExec64:  "Active fluentdwinsvc service is detected"
WixQuietExec64:  The service is starting or stopping.  Please try again later.
```

Sometimes the installation does not work and this may have something to do with it.
I will check it.

Reply:
I found the cause. 
At `InstallFluentdWinSvc` action, the service is started because of the option `--reg-winsvc-auto-start`, which is added here.
I think this makes the behavior unstable.

https://github.com/fluent/fluent-package-builder/blob/f285807b93b03c8338dd0b58371c247e6fbfd045/fluent-package/msi/source.wxs#L179-L181

We should disable this auto-start since few users want this auto-start.

* No one would be happy to start Fluentd with the default settings.
* It opens some ports by default. It is undesirable from a security point of view.

How about adding fix like this?: https://github.com/daipom/fluent-package-builder/commit/10ae6750565df7b1f70e0098404c8336565ced0a

I'm checking the CI results on my fork with this fix now.

Reply:
The CI on my fork succeeds!: https://github.com/daipom/fluent-package-builder/actions/runs/5504485369

Reply:
I confirmed the package built in the CI works as expected in my environment.

Reply:
It looks better to me, then could you push a commit on this pull request?

Reply:
In addition, please add description about removing `--reg-winsvc-auto-start` to the first comment.

Reply:
@kenhys Sure!

@ashie To the first comment of this PR?

Reply:
> To the first comment of this PR?

Yes, it's used as the commit message on squash merging.

Reply:
> > To the first comment of this PR?
> 
> Yes, it's used as the commit message on squash merging.

Instead of adding the change to this PR, I have made another PR:

* #521

I want to rebase this branch on it after it is merged.

Reply:
> It looks better to me, then could you push a commit on this pull request?

I have applied the changes except #521!

Reply:
Solved by #521.